docs/admin/etcd.md

```diff
@@ -0,0 +1,66 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/admin/etcd.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+# etcd
+
+[etcd](https://coreos.com/etcd/docs/2.0.12/) is a highly-available key value
+store for shared configuration and service discovery. It is used by Kubernetes
+for all cluster configuration data storage.
+
+## Configuration: high-level goals
+
+Access Control: give *only* kube-apiserver read/write access to etcd. You do not
+want apiserver's etcd exposed to every node in your cluster (or worse, to the
+internet at large), because access to etcd is equivilent to root in your
+cluster.
+
+Data Reliability: for reasonable safety, either etcd needs to be run as a
+cluster (multiple machines each running etcd) or etcd's data directory should be
+located on durable storage. (For example, GCE's persistent disk.) In either
+case, the data directory ought to be backed up periodically.
+
+## Default configuration
+
+The default setup scripts use kubelet's static file feature to run etcd in a
+[pod](../../cluster/saltbase/salt/etcd/etcd.manifest). This manifest should only
+be run on master VMs. The default location that kubelet scans for manifests is
+```/etc/kubernetes/manifests/```.
+
+## Clustered (HA) configuration
+
+See [Clustering etcd](high-availability.md#clustering-etcd) for an experimental
+way to do this.
+
+
```

Comment:
It would also be useful to say:
* objects are stored under `/registry` by default. This path can be prefixed with --etcd-prefix="/foo" on apiserver
* something about using etcdctl and hack/kube-dump.sh